tests20/python_client/testcases/test_index.py

```diff
@@ -72,9 +80,8 @@ def test_index_collection_None(self):
         """
         self._connect()
         index_name = cf.gen_unique_str(prefix)
-        ex, _ = self.index_wrap.index_init(None, default_field_name, default_index_params, name=index_name)
-        log.error(str(ex))
-        assert "invalid" or "illegal" in str(ex)
+        self.index_wrap.init_index(None, default_field_name, default_index_params, name=index_name, check_task=CheckTasks.err_res,
+                                           err_code=1, err_msg="Collection")
```

Comment:
we shall pass the check items and expectations into the check_items:
check_items={err_code: 1, err_msg: "blablabla..."}